Makefile

```diff
@@ -1,10 +1,10 @@
 default: install
 
 build:
-	go build
+	go build ./...
 
-install:
-	go install
+install: build
```

Comment:
I had install depending on build originally but it doubles the (admittedly quick) time to run `make install` so I removed it.